### DISCUSSION TITLE
Error generating forecast plan, seems to be getting more common?

### DISCUSSION THREAD

**Discussion by rustyshelf** [amberelectric/public-api#145] [General]
![IMG_25E1EEAF6215-1](https://github.com/amberelectric/public-api/assets/610217/e065e172-4b6b-4ca2-851b-d6f352d742c5)

I seem to be getting this more and more recently. Is it just me, or does this indicate a more general service outage? It's been like that for 5 minutes now, but I've also seen it intermittently quite a lot.

**Comment by madpilot** [comment]
Can you ping smartshift@amber.com.au, and I'll get someone to have a look
